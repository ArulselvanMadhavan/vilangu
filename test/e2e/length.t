int main() {
  int a[][][];
  a = new int[5][][];
  a[0] = new int[21333][];
  a[0][0] = new int[45];
  out a.length;
  out a[0].length;
  out a[0][0].length;
  out a[1].length;
  out a[1][0].length;
}