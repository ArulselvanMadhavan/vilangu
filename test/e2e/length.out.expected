5
21333
45
0
Array out of bounds exception at line 10. Index:0 | Length:0
